app/views/comments/_comments.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= t "comments.title" %>
+<% @post.comments.each do |comment| %>
+  <div class="well">
+    <p>
+      <strong>
+        <%= comment.username %>
+      </strong> :
+      <%= comment.body %>
+      <%= link_to t("delete.button"), [comment.post, comment], method: :delete, data:{confirm: t("confirm") } %>
+    </p>
+  </div>
+<% end %>
```

Comment:
You should create file '_comment.html.erb', then move code lines to that file, then call that file name back here
```suggestion
render "comment", comment: @post.comments
```